README.rst

```diff
@@ -126,10 +126,28 @@ A single ``else`` block may be present, but ``elif`` is not supported.
 
 See also `pyupgrade’s similar feature <https://github.com/asottile/pyupgrade/#python2-and-old-python3x-blocks>`__ that removes outdated code from checks on the Python version.
 
+Django 1.7
+-----------
```

Comment:
FYI, in reST the headers should match the text length exactly, I think if they don't match it can break PyPI

I should add a pre-commit check for this

anyway, I'm fixing

Reply:
Interesting, sry for that. I'm not familiar enough with the rst syntax. Kinda scary to be able to have trouble with pypi from that

Reply:
If you ever see a package on PyPI whose README looks like rst syntax rather than rendered, now you know

I use `twine check` at release time which catches this: https://twine.readthedocs.io/en/stable/#twine-check . But it would be good to have it caught on CI. One day I'll figure it out.